wikicurses/settings.py

```diff
@@ -1,16 +1,29 @@
+import sys
 import os
 import json
 import collections
 import configparser
+import shutil
 from wikicurses import formats
 from urllib.parse import urlparse
 
-default_configdir = os.environ['HOME'] + '/.config'
+# TODO: This may cause issues with python 2.7; however, as of 01OCT2019, it is nearing sunset
+default_configdir = os.path.expanduser('~') + '/.config'
 configpath = os.environ.get(
     'XDG_CONFIG_HOME', default_configdir) + '/wikicurses'
 
+configfile = os.path.join(configpath, "config")
+
+os.makedirs(configpath, exist_ok=True)
+
+if not os.path.isfile(configfile):
```

Comment:
Neat solution!

One thing I'm not so sure about: I think the current behavior using a global configuration file instead of copying like this might be better for disro packaging (rather than installing with `pip`). Arch Linux, at least, currently packages Wikicurses.

But then, maybe it would just complicate and confuse things too much to provide a mechanism for either copying or not copying...

I guess if we want this complexity, it might be best achieved by also checking for `/etc/wikicurses.conf`, and not copying the configuration if that exists.